index.html

```diff
@@ -925,6 +925,26 @@ <h2 id="docconformance">
               </p>
             </td>
           </tr>
+          <tr id="img-no-alt" tabindex="-1">
+            <td>
+              <code><a>img</a> with no <a data-cite=
+              "html/embedded-content.html#unknown-images">alt</a>=""</code>
```

Comment:
This doesn't look right... "alt" should link to the attribute... but if you want to link to the concept of "Images whose contents are not known", then that should be done properly (i.e., see my other comment about getting that exported from HTML).